internal/adapters/terraform/aws/rds/adapt_test.go

```diff
@@ -27,7 +27,7 @@ func Test_Adapt(t *testing.T) {
 				engine                  = "aurora-mysql"
 				backup_retention_period = 7
 				kms_key_id  = "kms_key_1"
-				storage_encrypted = true
+				storage_encrypted = false
```

Comment:
Why are these tests being changed?

Reply:
I changed it because I tested "aws rds describe-db-instances" on cli and  it gave me the stroage_encrypted value false. May be I didn't dug deep into it. if the defualt value is true or false